chaincode-labs/chaincode-podcast/privacy-and-robustness-in-lightning.md

```diff
@@ -458,84 +477,84 @@ And so, yeah, how do you push those to the front of the line and put them front
-So that is a very good philosophical question.
 
-Speaker 0: 00:22:45
+Rusty Russell: 00:22:45
 
+So that is a very good philosophical question.
 And hey, I work for a startup, right?
-And you should always be thinking what happens if my startup collapses because almost by definition right that's the the endgame for most startups is like they fail right and so when you have an open source project and you're like well I'm running this what what happens what happens after this to some extent the answer is you know I'm gonna keep working on this whatever happens unfortunately my wife has given explicit approval for me to take some time off and work.
+And you should always be thinking what happens if my startup collapses because almost by definition right that's the the endgame for most startups is like they fail right and so when you have an open source project and you're like well I'm running this what what happens what happens after this to some extent the answer is, I'm gonna keep working on this whatever happens unfortunately my wife has given explicit approval for me to take some time off and work.
 Should all of us fail?
 We've had this discussion, right?
 She's been like, cool, that's okay.
 You can go do that.
 At least for a while before you have to get a real job again.
-So I feel fairly confident to go, you know, we're going to move forward.
+So I feel fairly confident to go, we're going to move forward.
 And maybe not as fast as we are at the moment.
 But, you know, there's some future there.
 
-Speaker 2: 00:23:24
+Adam Jonas: 00:23:24
 
-Now, I guess my other bit of that was Because they're commercially associated, maybe not driven, commercially associated.
+I guess my other bit of that was Because they're commercially associated, maybe not driven, commercially associated.
 That means that features get pushed to the front of the line.
 Because I've worked for companies, I know how it works.
-So when you're talking about, you know, ring fencing things and making sure that we're, you know, thinking long term.
-These are different than just like building a product that's, you know, in JavaScript and you know, can wow an investor.
+So when you're talking about, ring fencing things and making sure that we're, thinking long term.
+These are different than just like building a product that's, in JavaScript and, can wow an investor.
 There is a philosophical side to what's happening here.
 And there is a collaboration side of no implementation can do it by themselves.
 
-Speaker 0: 00:24:07
+Rusty Russell: 00:24:07
 
 Yes.
 
-Speaker 2: 00:24:07
+Adam Jonas: 00:24:07
 
-And so those dynamics just don't seem to add up to a, a long term thinking and long term, you know, moving, moving the ball forward in these kinds of dimensions.
+And so those dynamics just don't seem to add up to a long term thinking and long term, moving, moving the ball forward in these kinds of dimensions.
 
-Speaker 0: 00:24:21
+Rusty Russell: 00:24:21
```

Comment:
Here you deleted the "you know" filler word, but you could also delete the duplicate phrase to make it more readable
```suggestion
And so those dynamics just don't seem to add up to a long term thinking and long term moving the ball forward in these kinds of dimensions.
```